portfolio/src/main/webapp/script.js

```diff
@@ -52,3 +52,14 @@ function addRandomTVQuote() {
   const tvQuoteContainer = document.getElementById('tvQuote-container');
   tvQuoteContainer.innerText = tvQuote;
 }
+
+/**
+ * Using async and await to fetch personal greeting data.
+ */
+async function getDataWithAsyncAwait() {
+  console.log('Fetching some data');
```

Comment:
I'm guessing you were using these logs for debugging, they don't look like they serve much purpose otherwise?
If everything is working you should remove your debugging artifacts unless they're also useful for production logs or something like that. (normally you'd have automated tests to prove everything is working, but apparently they aren't introducing you to the testing environment until week 5)

Reply:
Done